### PR TITLE
[12.0][MIG] document_quick_access

### PR DESCRIPTION
**Pull request by jarroyomorales**
Standard migration, no major changes.

@etobella

### CODE REVIEW

**Review by etobella — APPROVED** [review]
LGTM

**Comment by LoisRForgeFlow** [comment]
migration commit message is wrong :see_no_evil:

**Comment by jarroyomorales** [comment]
@LoisRForgeFlow Fixed! Thank you!

**Comment by jarroyomorales** [comment]
@LoisRForgeFlow Fixed! Thank you!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM!

**Review by JoanSForgeFlow — APPROVED** [review]
Functional review, LGTM :+1:

File: document_quick_access/models/base.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class Base(models.AbstractModel):
+    _inherit = 'base'
+
+    @api.multi
```
Comment (bealdav) [inline document_quick_access/models/base.py]:
```suggestion
```

**Comment by LoisRForgeFlow** [comment]
Still pending to remove `api` from imports.

**Comment by jarroyomorales** [comment]
@LoisRForgeFlow Forgot to remove it when attending @bealdav 's comment. Lets see if its green now

**Pull request merged**